components/datadog/apps/logger/images/logger/main_test.go

```diff
@@ -0,0 +1,222 @@
+package main
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Are you interested in running this on the CI? I can help! We have an integration test stage, we can add an unit test stage

Can also help migrating it to `testify`

Reply:
The app pretty simple and just there for some sanity checking. If it gets more complicated I'll definitely move to using `testify`.

> We have an integration test stage, we can add an unit test stage